subprojects/platform-native/src/testFixtures/groovy/org/gradle/nativeplatform/fixtures/app/IncrementalSwiftModifyExpectedOutputApp.groovy

```diff
@@ -19,23 +19,21 @@ package org.gradle.nativeplatform.fixtures.app
 /**
  * A Swift app with a changed source file
  */
-class IncrementalSwiftApp {
-    def app = new SwiftApp()
-    def alternateApp = new SwiftAlternateApp()
+class IncrementalSwiftModifyExpectedOutputApp extends IncrementalSwiftElement {
+    private final greeter = new SwiftGreeter()
+    private final sum = new SwiftSum()
+    private final main = new SwiftMain(greeter, sum)
+    private final alternateMain = new SwiftAlternateMain(greeter)
 
-    IncrementalSwiftApp() {
+    IncrementalSwiftModifyExpectedOutputApp() {
         // Verify some assumptions that the tests make
```

Comment:
Is this actually true?  Just glancing at the tests, this assumption did not jump out at me.